app/models/admin_ability.rb

```diff
@@ -302,5 +298,27 @@ def signed_in_with_track_organizer_role(user)
     can :toggle_user, Role do |role|
       role.resource_type == 'Track' && track_ids_for_track_organizer.include?(role.resource_id)
     end
+
+    # Show Events in the admin sidebar
+    can :update, Event do |event|
+      event.new_record? && conf_ids_for_track_organizer.include?(event.program.conference_id)
```

Comment:
We should look into rewriting this. Can update a new record does not sound very elegant.